msal/src/main/java/com/microsoft/identity/client/MultipleAccountPublicClientApplication.java

```diff
@@ -193,7 +193,7 @@ private void getAccountInternal(@NonNull final String identifier,
                                     @NonNull final String publicApiId) {
         try {
             validateNonNullArg(identifier, "identifier");
-            validateNonNullArg(identifier, "callback");
+            validateNonNullArg(callback, "callback");
         } catch (MsalArgumentException e) {
             callback.onError(e);
```

Comment:
If the callback is null, this catch block will throw an NPE

Reply:
so I think we can't do much here , than to throw a runtime exception like `IllegalArgumentException`